app/Exceptions/DockerContainerMissingException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+
+namespace App\Exceptions;
+
+
+use App\Shell\Shell;
+
+class DockerContainerMissingException extends \Exception
+{
+    public function render($request = null): void
+    {
+        $console = app('console');
+        $shell = app(Shell::class);
+
+        $console->line('');
+        $console->line($shell->formatErrorMessage('Docker container not found.'));
+    }
+}
```

Comment:
I think it would be great if we could give the user a more specific error message here, including the name of the container.

Reply:
Pushed up changes to include the container name if it is passed to the Exception